examples/main-launcher/src/main/scala/example/Main.scala

```diff
@@ -0,0 +1,14 @@
+package example
+
+import cats.effect.IO
+import tyrian.Html.*
+import tyrian.*
+
+object Main {
+  def main(args: Array[String]): Unit =
+    TyrianAppF.launchOnContentLoaded(Map(
```

Comment:
This needs to change slightly, people shouldn't be pulling in `TyrianAppF`, they should be using `TyrianApp` from the `IO` or `ZIO` modules. Maybe we just add a companion object there that delegates to this thing?

Reply:
I think also we could find a different name here to `launchOnContentLoaded`. :thinking: 

Is  `TyrianApp.onLoad(Map(..))` descriptive enough?

Would also be good to allow another constructor:

`def launchOnContentLoaded[F[_] : Async](appDirectory: (String, () => TyrianAppF[F, _, _]])*): Unit`

So that people can omit the `Map`:

```scala
TyrianAppF.launchOnContentLoaded(
  "CounterApp" -> () => CounterApp,
  "ChatApp" -> () => ChatApp
)
```

Reply:
I wonder if it's also possible to leave out the `thunk` syntax somehow? So you just get:

```scala
TyrianAppF.launchOnContentLoaded(
  "CounterApp" -> CounterApp,
  "ChatApp" -> ChatApp
)
```

What do you think?

Reply:
Ok, I have added the companion objects, changed the method name to `onLoad` with varargs and removed the thunk syntax.